gemini/src/main/java/com/techempower/cache/EntitySelector.java

```diff
@@ -26,14 +26,23 @@
   }
 
   /**
-   * Returns the entity of the selected class with the given id. Ignores any
+   * Returns the entity of the selected class with the given ids. Ignores any
    * specified method-value pairs.
    */
   public T get(long id)
   {
     return this.store.get(this.type, id);
   }
 
+  /**
+   * Returns the entity of the selected class with the given id. Ignores any
+   * specified method-value pairs.
+   */
```

Comment:
Very minor nit - these comments are backwards - this one takes in a list of ids and calls for `the given id` while the one above takes a single id but calls for `the given ids`.

Reply:
Whoops. Fixed.